server/lib/routes/post-metrics.js

```diff
@@ -77,20 +82,14 @@ function optionallyLogFlowBeginEvent (req, metrics, requestReceivedTime) {
           /*eslint-enable sorting/sort-object-props*/
         });
       }
-
-      return true;
     }
 
-    return false;
-  });
-
-  if (hasFlowBeginEvent) {
-    flowEvent('flow.begin', {
+    flowEvent(event.type, {
       flow_id: metrics.flowId, //eslint-disable-line camelcase
       flow_time: 0, //eslint-disable-line camelcase
       time: metrics.flowBeginTime
     }, req);
-  }
+  });
```

Comment:
I just was not sure if all flow events should be logged the same away, seems like we should just log all of them the same way (like flow.begin already was).